### PR TITLE
ALP-92: /history endpoint for fetching and updating last cleared time and last exported time

### LINKED ISSUE #92: Update AdminHistory table after export and clear actions

### PR DESCRIPTION
**Pull request by thaingocnguyen**

### CODE REVIEW

File: aunt-leahs-app/src/pages/AdminShiftDataPage.jsx [hunk 1]
```diff
@@ -41,17 +48,57 @@ const AdminShiftDataPage = () => {
...
 		catch (error) {
 			console.log("Error clearing shift data " + error);
 		}
 	}
 
+
+	async function exportShifts() {
+		try {
+			await axios.put('http://localhost:7071/api/history', {
+				isExportAction: 0,
```
Comment (michaelfromyeg) [inline aunt-leahs-app/src/pages/AdminShiftDataPage.jsx]: This should be 1, right?

File: aunt-leahs-functions/history/index.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    });
+
+    function getAdminHistory(tableName) {
+        queryString = 'SELECT lastClearedTime, lastExportedTime FROM AdminHistory WHERE name = @tableName';
+
+        request = new Request(queryString,
+            function (err) {
+                if (err) {
+                    context.log(err);
+                    context.done();
```
Comment (michaelfromyeg) [inline aunt-leahs-functions/history/index.js]: context.done(err)

File: aunt-leahs-functions/history/index.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        else {
+            queryString = 'UPDATE AdminHistory SET lastClearedTime = @editTime WHERE name = @tableName'
+        }
+
+        request = new Request(
+            queryString,
+            function (err) {
+                if (err) {
+                    context.log(err);
+                    context.done();
```
Comment (michaelfromyeg) [inline aunt-leahs-functions/history/index.js]: context.done(err)

**Review by michaelfromyeg — CHANGES_REQUESTED** [review]
Very minor comments, overall looks really good.

**Review by michaelfromyeg — APPROVED** [review]

**Comment by codygagnon** [comment]
Thai, can you please resolve the conflicts on this PR so that it can be approved and merged?

**Review by codygagnon — APPROVED** [review]

**Pull request merged**